### PR TITLE
Service mesh mentions removed

### PR DESCRIPTION
**Pull request by iArchitSharma**
I have removed some of the Service mesh mentions with Cloud native management, infrastructure etc

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 6c262f9e9c49a417f118ec9dda6ba71630060f27 at: https://656b9a24cb669e085b5bbfd5--meshery-docs-preview.netlify.app

File: docs/pages/concepts/architectural/meshsync.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Fingerprinting, the process of positively identifying and classifying resources,
 
 See Connnection State Management for additional information.
 
-Fingerprinting a service mesh is the act of uniquely identifying managed infrastructure, their versions and other specific characteristics.
+Fingerprinting is the act of uniquely identifying managed infrastructure, their versions and other specific characteristics.
```
Comment (leecalcote) [inline docs/pages/concepts/architectural/meshsync.md:67]: Good one.

File: docs/pages/guides/configuration-management.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ You can onboard ImageHub to an installed service mesh as shown below.
 
 You can also manage WASM filters through Meshery's CLI, mesheryctl.
 
-The `mesheryctl exp filter` subcommand lets you import and configure filters in your service meshes.
+The `mesheryctl exp filter` subcommand lets you import and configure filters.
```
Comment (leecalcote) [inline docs/pages/guides/configuration-management.md:85]:
```suggestion
The `mesheryctl filter` command lets you import and configure WebAssembly filters.
```

File: docs/pages/guides/meshery-metrics.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ In order to pull in these environment metrics, you can also manually configure M
     
   <section id="content3">
     <p>
-      Static Boards capture the Service Mesh Performance. Certain protos are defined and tracked as a part of performance tests. Static Boards Queries Prometheus SDK directly.
+      Static Boards capture the Cloud Native Performance. Certain protos are defined and tracked as a part of performance tests. Static Boards Queries Prometheus SDK directly.
```
Comment (leecalcote) [inline docs/pages/guides/meshery-metrics.md:47]:
```suggestion
      Static Boards capture the standard performance metrics included in every Meshery performance test irrespective of which load generator is used. A set calculations are made using statistical analysis of the metrics gathered in the static boards. Static Boards Queries Prometheus SDK directly.
```

File: docs/pages/guides/multiple-of-the-same-adapter.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ Alternatively, directly modify the `~/.meshery/meshery.yaml` configuration file,
 
 #### Configuration: Running more than one instance of the same Meshery adapter
 
-The default configuration of a Meshery deployment includes one instance of each of the Meshery adapters (that have reached a stable version status). You may choose to run multiple instances of the same type of Meshery adapter; e.g. two instances of the Meshery Adapter for NGINX Service Mesh. To do so, you can use either of Meshery's clients or to modify your Meshery deployment:
+The default configuration of a Meshery deployment includes one instance of each of the Meshery adapters (that have reached a stable version status). You may choose to run multiple instances of the same type of Meshery adapter; e.g. two instances of the Meshery Adapter for NGINX. To do so, you can use either of Meshery's clients or to modify your Meshery deployment:
```
Comment (leecalcote) [inline docs/pages/guides/multiple-of-the-same-adapter.md:29]: Please revert.

File: docs/pages/tasks/filter-management.md [hunk 5]
```diff
@@ -18,7 +18,7 @@ Meshery offers powerful features that simplify the management of Envoy WASM filt
 - **Publish and Clone Filters**: Share your filters with the community by publishing them in the [Meshery Catalog](https://meshery.io/catalog) and make it effortless for others to clone them.
 - **Download WASM Binaries**: Access and download WebAssembly binaries directly from your Remote Provider like [Layer5 Cloud](https://meshery.layer5.io/).
 - **Efficient Data Plane Design**: Seamlessly design and deploy Istio and Envoy data planes using extensions like [MeshMap](https://layer5.io/cloud-native-management/meshmap).
-- **Contribute to Open Source**: Get involved with the service mesh community by contributing to any of the 7 open-source [wasm-filters](https://github.com/layer5io/wasm-filters) developed by the Meshery community.
+- **Contribute to Open Source**: Get involved with the cloud native community by contributing to any of the 7 open-source [wasm-filters](https://github.com/layer5io/wasm-filters) developed by the Meshery community.
```
Comment (leecalcote) [inline docs/pages/tasks/filter-management.md:21]:
```suggestion
- **Contribute to Open Source**: Get involved with the cloud native community by contributing to any of the open source [wasm-filters](https://github.com/layer5io/wasm-filters) developed by the Meshery community.
```

**Comment by l5io** [comment]
🚀 Preview for commit 053190f5f4572c367e2a2aeb35f78f4c6470576a at: https://656c33a6c3408b5902eb2101--meshery-docs-preview.netlify.app

**Comment by iArchitSharma** [comment]
@leecalcote Done!

**Review by leecalcote — APPROVED** [review]

**Pull request merged**